client/packages/purchase_orders/src/api/hooks/usePurchaseOrder.ts

```diff
@@ -11,31 +10,18 @@ import {
   useTranslation,
   useUrlQuery,
   setNullableInput,
+  RecordPatch,
+  PurchaseOrderNodeStatus,
+  PurchaseOrderNodeType,
+  useDebounceCallback,
 } from '@openmsupply-client/common';
 import { usePurchaseOrderGraphQL } from '../usePurchaseOrderGraphQL';
 import { LIST, PURCHASE_ORDER } from './keys';
 import { PurchaseOrderFragment } from '../operations.generated';
-import { useMemo } from 'react';
+import { useMemo, useState, useEffect } from 'react';
 import { usePurchaseOrderColumns } from '../../DetailView/columns';
 
-export type UpdatePurchaseOrderInput = {
-  advancePaidDate?: string | null;
-  comment?: string | null;
-  confirmedDatetime?: string | null;
-  contractSignedDate?: string | null;
-  currencyId?: string | null;
-  donorId?: string | null;
-  foreignExchangeRate?: number | null;
-  id: string;
-  receivedAtPortDate?: string | null;
-  reference?: string | null;
-  requestedDeliveryDate?: string | null;
-  sentDatetime?: string | null;
-  shippingMethod?: string | null;
-  status?: PurchaseOrderNodeType | null;
-  supplierDiscountPercentage?: number | null;
-  supplierId?: string | null;
-};
```

Comment:
Removed to instead follow the pattern of using the Fragment generated by graphQL. See update input